### PR TITLE
kie-issues#1090: Move the source of `kie-sandbox-quarkus-accelerator` to `kie-tools` and use its repo as a deployment repo (Part 1)

### PR DESCRIPTION
**Pull request by tiagobento**
Part of https://github.com/apache/incubator-kie-tools/issues/3886

---

This PR introduces a new package (`@kie-tools/kie-sandbox-accelerator-quarkus` @ `packages/kie-sandbox-accelerator-quarkus`) with the source code that's currently in https://github.com/apache/incubator-kie-sandbox-quarkus-accelerator 0.0.0 branch.

Please go see README.md, DEV.md, and ARCHITECTURE.md on the new package for more details.

It also updates the release jobs to publish this source code to the https://github.com/apache/incubator-kie-sandbox-quarkus-accelerator repo, making it the "deployment" repo for the Quarkus Accelerator on KIE Sandbox, with development brought here to `kie-tools`.

This new package also contains a development facilitator that spawns an HTTP server to serve a Git bare repository. This is very useful during development.

I also took the opportunity to make development of KIE Sandbox itself a little bit easier, using the `concurrently` NPM package to spawn all services KIE Sandbox can integrate with. The Quarkus Accelerator included. 

You may notice that the Quarkus logo for the Quarkus Accelerator moved from the Accelerator itself to KIE Sandbox, making the Accelerator even more focused on being ready for users to start developing their Business Automation projects with BPMN and DMN.

Introducing a new parameter to the CORS proxy to use `http` for certain domains was also necessary, as the HTTP server for Git bare repos spawns locally without TLS.

Cosmetic changes were applied to the `index.html` for the CORS proxy too, resulting in tests adapted to it on `kie-sandbox-distribution`.

----

Extracting the HTTP server for Git bare repos to a separate package was something I considered doing in this PR, but I figured I could leave it for when we actually need it for other use-cases, like the Samples or the Accelerators for Serverless Logic Web Tools (packages/serverless-logic-web-tools). cc @fantonangeli @ricardozanini ;)

### CODE REVIEW

File: packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+//
+// main
+
+const serveAsStaticContent = __serveStatic(contentRoot);
+
+__http
+  .createServer((req, res) => {
+    // bare git repos
+    if (req.url?.split("/")[1].endsWith(".git")) {
+      console.log(`[git-repo-http-dev-server] Received request for '${req.url}'...`);
```
Comment (fantonangeli) [inline packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs:60]: There are some lines like:
```
console.log(`[git-repo-http-dev-server] MESSAGE`);
```
Can we make a simple function to call `console.log` with the "formatted" message?

**Review by fantonangeli — CHANGES_REQUESTED** [review]
LGTM, I left a xxs suggestion

File: packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs (hunk 1 above)
Comment (tiagobento) [reply packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs:60]: Not exactly sure what you mean... each call to `console.log` is a different log entry...

Comment (fantonangeli) [reply packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs:60]: I mean to call a function like:
```ts
function logger(message: string){
 console.log(`[git-repo-http-dev-server] ${message}`);
}
```
So we can easily change all the messages if we need. 
Wdyt?

Comment (tiagobento) [reply packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs:60]: Not a big fan of the indirection, as the `logger` function would always force `console.log`. Finding and replacing works well and everything is contained to one single file...

Comment (tiagobento) [reply packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs:60]: If this ever gets more complicated, maybe it's worth refactoring, but not in the current state.

Comment (fantonangeli) [reply packages/kie-sandbox-accelerator-quarkus/dev-server/server.mjs:60]: perfectly fine for me

**Review by fantonangeli — APPROVED** [review]

**Review by ljmotta — APPROVED** [review]
@tiagobento thanks for this PR. LGTM!

**Pull request merged**